CHANGELOG.md

```diff
@@ -6,6 +6,31 @@
 
 **Added**:
 
+- **decidim-conferences**: Added Conferences as a Participatory Space. This module is a configurator and generator of Conference pages, understood as a collection of Meeting. [\#3781](https://github.com/decidim/decidim/pull/3781)
+- **decidim-admin**:Add link to user profile and link to conversation from admin space. [\#3995](https://github.com/decidim/decidim/pull/3995)
```

Comment:
The CHANGELOG is adding too many things due to a rebase, can you fix it so we can merge this please? 😄